g3doc/quick_reference.md

```diff
@@ -403,6 +406,29 @@ time-critical code:
     Potentially slow, it may be better set all elements of an aligned array and
     then `Load` it.
 
+### Getting/setting blocks
+
+*   <code>Vec<BlockDFromD<DFromV<V>>> **ExtractBlock**&lt;int kBlock&gt;(V)
+    </code>: returns block `kBlock` of V, where `kBlock` is an index to a block
+    that is `HWY_MIN(DFromV<V>().MaxBytes(), 16)` bytes.
+
+    `kBlock` must be in `[0, DFromV<V>().MaxBlocks())`.
+
+*   <code>V **InsertBlock**&lt;int kBlock&gt;(V v, Vec<BlockDFromD<DFromV<V>>>
+    blk_to_insert)</code>: Inserts `blk_to_insert`, with `blk_to_insert[i]`
+    inserted into lane `kBlock * (16 / sizeof(TFromV<V>)) + i` of the result
+    vector, if `kBlock * 16 < Lanes(DFromV<V>()) * sizeof(TFromV<V>)` is true.
+
+    Otherwise, returns `v` if `kBlock * 16` is greater than or equal to
+    `Lanes(DFromV<V>()) * sizeof(TFromV<V>)`.
+
+    `kBlock` must be in `[0, DFromV<V>().MaxBlocks())`.
+
+*   <code>size_t **NumOfBlocks**(D d)</code>: Returns the number of 16-byte
```

Comment:
Would you mind renaming this to Blocks(d) for consistency with Lanes(d)?

Reply:
Made the changes